Inc/module_list.h

```diff
@@ -33,6 +33,7 @@ typedef enum {
     WRITE_ALIAS, // Get and save a new given alias.
     ASK_PUB_CMD, // Gate asks a sensor module to publish its data
     REVISION, // Module sends its firmware revision
+    LUOS_REVISION, // Module sends its luos revision
```

Comment:
Whatch out of the position of your new message command. If you put it here you create a decay on all next messages and broke the compatibility with older revision of Luos.
To make it work put it ad the end of the list, there is a sectiion dedicated to it.